input/omnia_config.yml

```diff
@@ -68,3 +68,9 @@ k8s_service_addresses: "10.233.0.0/18"
 # This network must be unused in your network infrastructure.
 # Default value is "10.233.64.0/18"
 k8s_pod_network_cidr: "10.233.64.0/18"
+
+#----------------------------VERIFY INTEL GAUDI INSTALLATION------------------------------------------------------
+
+# This variable controls running hardware sanity test only or not, when verify Intel Gaudi installation with hl_qual.
+# Set this to false if you want to run the complete tests.
+verify_intel_gaudi_with_sanity_test_only: false
```

Comment:
Can we remove this variable from omnia_config.yml. We will have individual utility only for hardware sanity test.

Reply:
removed

Reply:
@abhishek-sa1  should we restore this change?

Reply:
yes..we can restore this change

Reply:
done

Reply:
by default it will be false right?

Reply:
no  true by default ie only sanity test are run